### PR TITLE
Hash and Sign

### LINKED ISSUE #45: Implement Hash and Sign for IDs and Attestations
Use a library containing SHA256 for hashing IDs and use a library containing ED25519 for signing the Attestations.

### PR DESCRIPTION
**Pull request by celinecamacho**
Closes #45

### CODE REVIEW

**Review by lhmerino — COMMENTED** [review]
Very nice! Signature algorithm looks spot on.

**Review by jbmichel14 — APPROVED** [review]
Nice work

File: app/src/test/java/com/github/dedis/student20_pop/model/ElectionTest.java [hunk 1]
```diff
@@ -59,9 +62,13 @@ public void getOptionsTest() {
         assertThat(election1.getOptions(), is(options));
     }
 
+    @Ignore("Need the private key of the organizer, will test later")
```
Comment (AlexisTabin) [inline app/src/test/java/com/github/dedis/student20_pop/model/ElectionTest.java:65]: Maybe add a TODO comment so we don't forget to do this test

File: app/src/test/java/com/github/dedis/student20_pop/model/EventsTest.java [hunk 2]
```diff
@@ -73,10 +75,14 @@ public void getTypeTest() {
         assertThat(event1.getType(), is(type));
     }
 
+    @Ignore("Need the private key of the organizer, will test later")
```
Comment (AlexisTabin) [inline app/src/test/java/com/github/dedis/student20_pop/model/EventsTest.java:78]: same here

**Review by AlexisTabin — APPROVED** [review]

**Pull request merged**